Exiled.Events/EventArgs/Item/ChangingAmmoEventArgs.cs

```diff
@@ -43,20 +43,20 @@ public ChangingAmmoEventArgs(InventorySystem.Items.ItemBase firearm, byte oldAmm
         public Player Player { get; }
 
         /// <summary>
-        /// Gets the <see cref="API.Features.Items.Firearm"/> the ammo is being changed to.
+        /// Gets the <see cref="API.Features.Items.Firearm"/> the ammo type is being changed to.
```

Comment:
I think here is ammo value changing but not type

Reply:
It looks like it ties into a firearm, the documentation on this page as a whole is rather poor so it can be ammo count or ammo type, its not very clear